CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 - Deployed `L2BalancerPseudoMinter` to all networks except Ethereum.
 - Deployed `VotingEscrowDelegationProxy` and `NullVotingEscrow` to all networks except Ethereum.
 - Deployed `WeightedPoolFactory` V4 to all networks.
-- Deployed `ComposableStablePoolFactory` V4 to all networks.
+- Deployed `ComposableStablePoolFactory` V5 to all networks.
```

Comment:
We did use V4, and have marked it deprecated. Shouldn't this retain V4 and add V5 to the end? It's a bit weird to lump two deployments into one release, but it also looks weird to have it deprecated below and not mentioned above.

Reply:
Sounds good; addressed in https://github.com/balancer/balancer-deployments/pull/61/commits/6a43fd278a48acdce6320b863d97e5db4de4a007.